compute_endpoint/tests/unit/test_endpointmanager_unit.py

```diff
@@ -252,26 +259,34 @@ def epmanager_as_root(mocker, conf_dir, mock_conf, mock_client, mock_pim):
     em.request_shutdown(None, None)
 
 
+@pytest.fixture
+def ident():
```

Comment:
I suppose this is a normal/accepted way of referencing the same constant in multiple test files, which is interesting, instead of a ``MOCK_IDENT = "successful_exec_some_identity"`` line.  But I'm sure 'reasons'.

Reply:
In this case, either-or.  Use of the fixture makes it more "formal" in my mind, but "meh."